### PR TITLE
Add code to load current user id

### PR DESCRIPTION
**Pull request by TSHEGOFATSOMojela**
#### Related issues

- connect to <link_to_referenced_issue>

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: client/js/app.js [hunk 1]
```diff
@@ -13,4 +13,15 @@ angular
       });
 
     $urlRouterProvider.otherwise('todo');
-  }]);
+  }])
+  .run([
+    '$rootScope', 'LoopBackAuth', 'User',
+    function($rootScope, LoopBackAuth, User) {
+      $rootScope.$on('$stateChangeStart', function (event, toState, toParams) {
+        //var currentUserId = LoopBackAuth.load('currentUserId');
```
Comment (b-admike) [inline client/js/app.js]: is this meant to be commented out?

Comment (bajtos) [reply client/js/app.js]: TBH I have no idea what is this about :(

**Comment by kjdelisle** [comment]
Closing this PR since it appears to be specific to your usage, and no explanation was given for its content.

**Pull request closed without merging**